doc/elements.md

```diff
@@ -1,16 +1,93 @@
+- [`pl-checkbox`](#pl-checkbox-element): Selecting **multiple options** from a
+  list.
+- [`pl-number-input`](#pl-number-input-element): Fill in a **numerical** value 
+  within a specific tolerance level such as 3.14, -1.921, and so on.
+- [`pl-integer-input`](#pl-integer-input-element): Fill in an **integer** value 
+  such as -71, 0, 5, 21, and so on.
+- [`pl-symbolic-input`](#pl-symbolic-input-element): Fill in a **symbolic** value
+  such as `x^2`, `sin(z)`, `mc^2`, and so on.
+- [`pl-string-input`](#pl-string-input-element): Fill in a **string** value
+  such as "Illinois", "GATTACA", "computer", and so on.
+- [`pl-matrix-component-input`](#pl-matrix-component-input-element): Fill in
+  a **matrix** using grid that has an input area for each element.
+- [`pl-matrix-input`](#pl-matrix-input-element): Supply a matrix in a supported
+  programming language format.
+- [`pl-file-editor`](#pl-file-editor-element): Provide an in-browser code editor
+  for writing and submitting code.
+- [`pl-file-upload`](#pl-file-upload-element): Provide a submission area
+  to obtain a file with a specific naming scheme.
+- [`pl-threejs`](#pl-threejs-element): Enables 3D scene display and problem
+  submission.
+
+**Decorative** elements are meant to improve how the question is displayed to
+students. Elements under this category include ways to specify question markup,
+images, files, and code display. The following **decorative** elements are available:
+
+- [`pl-code`](#pl-code-element): Displays code rendered with the appropriate
+  syntax highlighting. 
+- [`pl-figure`](#pl-figure-element): Embed an image file in the question.
+- [`pl-file-download`](#pl-file-download-element): Enable file downloads for
+  data-centric questions.
+- [`pl-variable-output`](#pl-variable-output-element): Displays matrices in 
+  code form for supported programming languages.
+- [`pl-matrix-latex`](#pl-matrix-latex-element): Displays matrices using 
+  appropriate LaTeX commands for use in a mathematical expression.
+- [`pl-question-panel`](#pl-question-panel-element): Displays the text of a question.
+- [`pl-submission-panel`](#pl-submission-panel-element): Displays the answer 
+  given by the student.
+- [`pl-answer-panel`](#pl-answer-panel-element): Displays the correct
+  answer to a given question.
+- [`pl-external-grader-results`](#pl-external-grader-results-element):
+  Displays results from questions that are externally graded.
+- [`pl-prairiedraw-figure`](#pl-prairiedraw-figure-element): Show a PrairieDraw
+  figure.
+- [`pl-graphviz-render`](#pl-graphviz-render-element): Show a Graphviz DOT 
+  figure.
+
+Note: PrairieLearn Elements listed next have been **deprecated**. These elements
+will be removed at a future date.
+
+- [`pl-variable-score`](#pl-variable-score-element): Displays a partial score
+  for a submitted element.
+    - **Deprecated** as submission elements in `v3` all have score display options.
+
+## Submission Elements --
```

Comment:
The double dash after this isn't really adding much and can probably be removed.

Reply:
Done.